fr/integrations/plugin-packs/procedures/hardware-storage-lenovo-iomega-snmp.md

```diff
@@ -0,0 +1,182 @@
+---
+id: hardware-storage-lenovo-iomega-snmp
+title: Lenovo Iomega
+---
+
+## Vue d'ensemble
+
+La solution de stockage réseau Lenovo Iomega StoreCenter (ix2) offre sécurité, stockage et partage de contenu avancés dans un produit simple d'emploi et idéal pour les petits réseaux. 
+	
+## Contenu du Pack de supervision
+
+### Objets supervisés
+
+* Lenovo Iomega (ix2)
+* Hardware
+* Stockage
+* NAS
```

Comment:
Could you add the list of the service template ? 'Cause we now that we are going to monitor Lenovo Iomega :)

Reply:
It's done ^^